vendor/bat-native-ledger/src/bat/ledger/internal/legacy/client_properties.cc

```diff
@@ -47,4 +67,133 @@ bool ClientProperties::operator!=(
   return !(*this == rhs);
 }
 
+base::Value::Dict ClientProperties::ToValue() const {
+  base::Value::Dict dict;
+  dict.Set(kWalletInfoKey, wallet_info.ToValue());
+  dict.Set(kBootTimestampKey, base::Int64ToValue(boot_timestamp));
+  dict.Set(kReconcileTimestampKey, base::Int64ToValue(reconcile_timestamp));
+  dict.Set(kFeeAmountKey, fee_amount);
+  dict.Set(kUserChangedFeeKey, user_changed_fee);
+  dict.Set(kRewardsEnabledKey, rewards_enabled);
+  dict.Set(kAutoContributeKey, auto_contribute);
+
+  base::Value::Dict inline_tips_dict;
+  for (const auto& inline_tip : inline_tips) {
+    inline_tips_dict.Set(inline_tip.first, inline_tip.second);
+  }
```

Comment:
```
for (const auto& [key, value] : inline_tips) {
  inline_tips_dict.Set(key, value);
}
```

Reply:
Fixed in bd27c663d1e5553c0b4adc67d75f363974e71300.